woocommerce/woocommerce
# [Settings] Add Sidebar Items

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Partially addresses https://github.com/woocommerce/woocommerce/issues/52275

This PR adds Sidebar items and updates styles to match.

This PR does not handle routing, links, or page specific into such as page title in the sidebar. These items will be addressed in a follow up when https://github.com/woocommerce/woocommerce/pull/52692 is merged.

<img width="1028" alt="image" src="https://github.com/user-attachments/assets/06aaa10f-1b8c-462d-90d4-bc9f84f33733">



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Visit JS site and turn on the feature flag `settings`, Tools > WCA Test Helper > Features.
2. Install/Activate Gutenberg.
3. Visit the new Settings, WooCommerce > Settings.
4. See the new layout and sidebar populated with items.


### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

Updates unreleased feature

</details>
